_layouts/partners/faq.html

```diff
@@ -0,0 +1,129 @@
+                    <div id="logistics-section" class="section-title-left">{{ page.logistics_section_title | markdownify}}</div>
+                    <button
+                      class="section-title-right collapse-expand-btn-logistics usa-button usa-button--unstyled"
+                      onclick="javascript:updateAllAccordions('logistics')">
+                      Expand all
+                    </button>
+                  </div>
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5">
+                      {% for item in page.logistics_accordion %}
+                        <h3 class="usa-accordion__heading">
+                          <button
+                            onclick="updateSingleAccordion(this, 'logistics')"
+                            class="usa-accordion__button login-accordion-btn-logistics"
+                            aria-expanded="false"
+                            aria-controls={{ item.id }}
+                            >
+                            {{ item.title }}
+                          </button>
+                        </h3>
+                        <div id={{ item.id }} class="usa-accordion__content login-accordion-content-logistics">
+                        {{ item.content | markdownify}}
+                        </div>
+                      {% endfor %}
+                  </div>
+                  <a href="#main-content">Back to Top</a>
+                  <div class="section-title-row margin-top-5">
+                    <div id="development-section" class="section-title-left">{{ page.development_section_title | markdownify}}</div>
+                    <button
+                      class="section-title-right collapse-expand-btn-development usa-button usa-button--unstyled"
+                      onclick="javascript:updateAllAccordions('development')">
+                      Expand all
+                    </button>
+                  </div>
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5">
+                      {% for item in page.development_accordion %}
+                        <h3 class="usa-accordion__heading">
+                          <button
+                            onclick="updateSingleAccordion(this, 'development')"
+                            class="usa-accordion__button login-accordion-btn-development"
+                            aria-expanded="false"
+                            aria-controls={{ item.id }}
+                            >
+                            {{ item.title }}
+                          </button>
+                        </h3>
+                        <div id={{ item.id }} class="usa-accordion__content login-accordion-content-development">
+                        {{ item.content | markdownify}}
+                        </div>
+                      {% endfor %}
+                  </div>
+                  <a href="#main-content">Back to Top</a>
```

Comment:
Same.

```suggestion
                  <a href="#main-content" class="usa-link">Back to Top</a>
```

(Not a blocker for initial merge)

Reply:
Filed as [LG-6404](https://cm-jira.usa.gov/browse/LG-6404)